PowerEditor/src/Parameters.h

```diff
@@ -835,6 +835,7 @@ struct NppGUI final
 	bool _backSlashIsEscapeCharacterForSql = true;
 	bool _stopFillingFindField = false;
 	bool _monospacedFontFindDlg = false;
+	bool _useDirectWrite = false;

```

Comment:
There are 4 different possible values (0-3) and users could ask for adding 2 & 3 in the future.
So using an enum variable `_writeTechnologyEngine` with 2 values (0 & 1) might preserve the settings interface in `config.xml`. So if there's any change in  the future, we just need to change the GUI without touching the `config.xml`